packages/dashboard/src/vaadin-dashboard-layout-mixin.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2024 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ *
+ * See https://vaadin.com/commercial-license-and-service-terms for the full
+ * license.
+ */
+import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+/**
+ * A mixin to enable the dashboard layout functionality
+ *
+ * @polymerMixin
+ * @mixes ResizeMixin
```

Comment:
There is no `ResizeMixin` used here, so this line should be probably removed.

Reply:
Thanks, removed for now. ResizeMixin is to be added in a separate PR.